### DISCUSSION TITLE
zh/reference/get_started/install

### DISCUSSION THREAD

**Discussion by giscus** [EdurtIO/incubator-dbm#171] [Q&A]
# zh/reference/get_started/install

Database Gui!

https://dbm.incubator.edurt.io/zh/reference/get_started/install.html

**Comment by 24kGarry** [comment]
npm install 报错误RequestError: connect ETIMEDOUT 20.205.243.166:443，

通过以下方式镜像加速：
npm config set ELECTRON_MIRROR https://npmmirror.com/mirrors/electron/
yarn config set electron_mirror https://npmmirror.com/mirrors/electron/
